content/en/blog/2025/sig-etcd-spotlight/sig-etcd-spotlight.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: blog
+title: "Spotlight on SIG etcd"
+slug: sig-etcd-spotlight
+date: 2025-02-05
+author: "Frederico Muñoz (SAS Institute)"
+---
+
+In this SIG etcd spotlight we talked with [James Blair](https://github.com/jmhbnz), [Marek
+Siarkowicz](https://github.com/serathius), [Wenjia Zhang](https://github.com/wenjiaswe), and
+[Benjamin Wang](https://github.com/ahrtr) to learn a bit more about this Kubernetes Special Interest
+Group.
+
+## Introducing SIG etcd
+
+**Frederico: Hello, thank you for the time! Let’s start with some introductions, could you tell us a
+bit about yourself, your role and how you got involved in Kubernetes.**
+
+**Benjamin:** Hello, I am Benjamin. I am a SIG etcd Tech Lead and one of the etcd maintainers. I
+work for VMware, which is part of the Broadcom group. I got involved in Kubernetes & etcd & CSI
```

Comment:
etcd?

Reply:
That's the normal capitalization, yes.

Reply:
Not sure what I thought was wrong here. Resolved.

Reply:
Yes, I've consistently changed everything to "SIG etcd", I initially had 3 different spellings.